package.json

```diff
@@ -211,6 +211,7 @@
     "mobx-react": "^5.0.0",
     "moment": "^2.22.0",
     "normalize.css": "^8.0.0",
+    "npm": "^6.4.1",
```

Comment:
Do we need npm intalled locally for something?

Reply:
no! nice catch - i probably ran npm update without the `-g` flag 🙈